model/private_subnet.rb

```diff
@@ -43,7 +43,7 @@ def self.random_subnet
   end
 
   def random_private_ipv4
-    addr = net4.nth_subnet(32, SecureRandom.random_number(2**(32 - net4.netmask.prefix_len) - 1))
+    addr = net4.nth_subnet(32, SecureRandom.random_number(2**(32 - net4.netmask.prefix_len) - 1).to_i + 1)
```

Comment:
What happen if SecureRandom return last subnet? +1 causes any issue?

Reply:
That's actually right, last subnet shouldn't be allowed, too. For example, `nth_subnet(32, 64)` returns `nil`. Therefor, we should have 63 possible ip addresses, first one is not allowed, and the index can be 63 at max. I fixes it accordingly.